miniwallet/test/utilities/index.ts

```diff
@@ -3,6 +3,10 @@ import { expect } from 'chai'
 import { MockProvider } from 'ethereum-waffle'
 import { Contract, BigNumber } from 'ethers'
 import { ethers } from 'hardhat'
+// TODO Parameterize CallData
+const ContractPath = '../../build/contracts/miniWallet/miniWallet.sol/MiniWallet.json'
```

Comment:
should merge into one line

Reply:
in fact it is not needed since you only need signature for initialize function

Reply:
This can be covered as stated [here](https://github.com/polymorpher/sms-wallet/blob/ws-miniwallet-v0.1/miniwallet/test/utilities/index.ts#L6)
```
// TODO Parameterize CallData
```

The calldata includes both the signature for the initialize function as well as the parameters see [here](https://github.com/polymorpher/sms-wallet/blob/ws-miniwallet-v0.1/miniwallet/test/utilities/index.ts#L42) which is using the passed paramaters from [here](https://github.com/polymorpher/sms-wallet/blob/ws-miniwallet-v0.1/miniwallet/test/approve.ts#L28)
i.e.
```
        [
          config.test.miniWallet.initialOperatorThreshold,
          config.test.miniWallet.initialOperators,
          config.test.miniWallet.initialUserLimit,
          config.test.miniWallet.initialAuthLimit
        ]
```

Reply:
Have changed this to use the following line 

`    const calldata = contract[1].interface.encodeFunctionData('initialize', contract[2] || [])`

Note: all Proxy contracts we are deploying will use the `initialize` function so can hardcode this for now.

Reply:
okay